llama2.mojo

```diff
@@ -431,10 +430,10 @@ struct TransformerWeights:
 
 
 fn read_file(file_name: String, inout buf: FileBuf) raises:
-    let _os = Python.import_module("os")
-    let ff_size = _os.path.getsize(file_name)
-    let cp_size = string.atol(ff_size.to_string())
-    let cp_buf: BufferPtrType = BufferPtrType.alloc(cp_size)
+    var ff = open(file_name, "r")
+    let ff_size = len(ff.read())
```

Comment:
omg.. it read full file into memory just to get it's size.. imagine huge checkpoint like llama7b -- 14GB of data will be loaded, and on some hosts it can fail

Reply:
haha I just realized yeah thats not good... any idea how we can do this without the external os library?